### ISSUE TITLE
NativeScript-UI Crashing bug on Refresh

### ISSUE THREAD

**Issue by NathanaelA** [telerik/nativescript-ui-feedback#1092]
### Tell us about the problem
In the Refresh function:
```
 RadListView.prototype.refresh = function () {
        this.clearEmbeddedViews();
        this.recreateEmbeddedViews();
        this.loadData();

        this.listViewLayout.forceRefreshLayout();
    };
```
Calling a `refresh()` very early in the loading/creating cycle will crash because `this.listViewLayout` is currently undefined, which will cause:
`System.err: TypeError: Cannot read property 'forceRefreshLayout' of undefined`

I have a plugin (nativescript-orientation) that calls refresh during certain events; unfortunately this event is called during the navigation to make sure the CSS changes are propagated properly to all children.  

### Which platform(s) does your issue occur on?
Android

### Please provide the following version numbers that your issue occurs with:
"nativescript-ui-listview": "6.0.0",

**Comment by tsonevn** [comment]
Hi @NathanaelA,
This issue is fixed with the latest  `nativescript-ui-listview` v.6.2.0.

**Comment by NathanaelA** [comment]
@tsonevn  - You are correct; this is already fixed in 6.2.0.   Tested and verified.  Thanks.
